docs: expand on statement to make clearer

udohjeremiah opened the pull request:
The current explanation isn't clear, as it uses ideas that require the reader to map meaning onto. So it's best to expound on it.

udohjeremiah committed:
Expand on statement to make clearer

The current explanation isn't clear, as it uses ideas that require the reader to map meaning onto. So it's best to expound on it.

Signed-off-by: Udoh Jeremiah <udohjeremiah@icloud.com>

udohjeremiah committed:
doc: remove misleading word

In Fastify, encapsulation is a good and encouraged behaviour. So, to use the word **"avoid"** here sounds misleading and negative.

Signed-off-by: Udoh Jeremiah <udohjeremiah@icloud.com>

Fdawgs reviewed (changes requested):
The original sentence should be rewritten for clarity rather than bolting on an extra sentence to repeat points.

udohjeremiah committed:
docs: clarify explanation of plugin encapsulation scope

This update rephrases the explanation of encapsulation to improve clarity and precision.

Signed-off-by: Udoh Jeremiah <udohjeremiah@icloud.com>

udohjeremiah committed:
Merge branch 'fastify:main' into patch-1

udohjeremiah committed:
Merge pull request #1 from udohjeremiah/patch-1

Merge PR

udohjeremiah committed:
docs: revert changes

Signed-off-by: Udoh Jeremiah <udohjeremiah@icloud.com>
